.github/workflows/fmt.yaml

```
@@ -7,30 +7,42 @@ on:
-      - name: Lint code with cargo clippy
-        run: cargo clippy -- -D warnings
+        uses: dtolnay/rust-toolchain@v1
+
+      - name: Run quality checks
+        run: |
+          set -euo pipefail
+          if [ "${{ matrix.check }}" == "cargo_fmt" ]; then
+            cargo fmt -- --check
+          elif [ "${{ matrix.check }}" == "cargo_clippy" ]; then
+            cargo clippy -- -D warnings
+          else 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)